### PR TITLE
Misc admin portal fixes

### PR DESCRIPTION
**Pull request by ellis-johnson**
### Which issue this PR addresses:

Fixes
https://issues.redhat.com/browse/ARO-1771
https://issues.redhat.com/browse/ARO-1770
https://issues.redhat.com/browse/ARO-1887

### What this PR does / why we need it:

Miscelaneous Admin Portal Fixes Including
- Security Fixes for npm libraries
- Support for prometheus graph in 4.11
- Updated redirect when cookie expires
- Update kubernetes logo for kube.config

And more

### Test plan for issue:

unit tests added where applicable

### Is there any documentation that needs to be updated for this PR?

No

### CODE REVIEW

**Review by tsatam — APPROVED** [review]
reviewed, lgtm!

Is there any information on the intent and desired scope for showing ClusterOperator information in the admin portal (Jira or otherwise)? I'm curious if there are any considerations I should make for consumption in the admin portal when adding controller statuses to the ARO operator: https://issues.redhat.com/browse/ARO-1628

File: pkg/portal/prometheus/prometheus.go [hunk 1]
```diff
@@ -48,10 +48,14 @@ func New(baseLog *logrus.Entry,
 	}
 
 	aadAuthenticatedRouter.NewRoute().Path("/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/microsoft.redhatopenshift/openshiftclusters/{resourceName}/prometheus").HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		r.URL.Path += "/"
+		r.URL.Path += "/graph"
```
Comment (darthhexx) [inline pkg/portal/prometheus/prometheus.go:51]: This is changing all Prometheus routes to `/graph`, but it has to be based on the cluster version. Only 4.11 requires the appended `/graph`, so all other clusters in the fleet won't work with this blanket change.

**Review by darthhexx — DISMISSED** [review]
The prometheus URL change needs to be refactoedto take into acount older OCP versions. 

Also, it would be optimal to increase the size of the cluster menu so that "ClusterOperators" displays fully. In the cluster operator individual status blade, could we look at a more organised layout. If you take a look at cloud-controller-manager, you'll see what I mean. Maybe a table layout or just all one under the other?

File: pkg/portal/cluster/clusteroperators.go [hunk 2]
```diff
@@ -27,15 +36,25 @@ func clusterOperatorsInformationFromOperatorList(operators *configv1.ClusterOper
...
 		for _, cnd := range co.Status.Conditions {
+			var condition OperatorCondition
 			if cnd.Type == "Available" {
 				Available = cnd.Status
 			}
+			condition.Message = cnd.Message
+			condition.Reason = cnd.Reason
+			condition.Status = cnd.Status
+			condition.LastUpdated = cnd.LastTransitionTime.String()
+			condition.Type = cnd.Type
```
Comment (facchettos) [inline pkg/portal/cluster/clusteroperators.go:49]: could we transform that into a struct instantiation? 
```suggestion
			if cnd.Type == "Available" {
				Available = cnd.Status
			}
			condition := OperatorCondition{
				Message: cnd.Message,
				....
			}
```

File: pkg/portal/cluster/clusteroperators.go [hunk 3]
```diff
@@ -27,15 +36,25 @@ func clusterOperatorsInformationFromOperatorList(operators *configv1.ClusterOper
 	for _, co := range operators.Items {
 		var Available = configv1.ConditionUnknown
```
Comment (facchettos) [inline pkg/portal/cluster/clusteroperators.go:37]: not from this pr, but I'm not sure why this is capitalized, that would mean that it is exported and is kind of confusing

File: pkg/portal/cluster/clusteroperators_test.go [hunk 4]
```diff
@@ -55,33 +56,173 @@ func TestClusterOperators(t *testing.T) {
...
 
 	sort.SliceStable(info.Operators, func(i, j int) bool { return info.Operators[i].Name < info.Operators[j].Name })
 	sort.SliceStable(expected.Operators, func(i, j int) bool { return expected.Operators[i].Name < expected.Operators[j].Name })
 
+	layout := "2006-01-02 15:04:05 -0700 MST"
+	for i, r := range info.Operators {
+		for j, c := range r.Conditions {
+			_, err := time.Parse(layout, c.LastUpdated)
+			if err != nil {
+				t.Error("Given Date was not in the correct format " + layout + err.Error())
```
Comment (facchettos) [inline pkg/portal/cluster/clusteroperators_test.go:219]: case is a little weird here

**Review by facchettos — CHANGES_REQUESTED** [review]

**Comment by ellis-johnson** [comment]
> reviewed, lgtm!
> 
> Is there any information on the intent and desired scope for showing ClusterOperator information in the admin portal (Jira or otherwise)? I'm curious if there are any considerations I should make for consumption in the admin portal when adding controller statuses to the ARO operator: https://issues.redhat.com/browse/ARO-1628

The intent currently is just to show the same sort of information that we would see through the oc command, so just as usually expected I believe

File: pkg/portal/cluster/clusteroperators.go (hunk 2 above)
Comment (ellis-johnson) [reply pkg/portal/cluster/clusteroperators.go]: @facchettos I accidentally merged these changes into this PR. All cluster opertor admin portal changes are on this PR https://github.com/Azure/ARO-RP/pull/2587. I will address these comments there

**Comment by darthhexx** [comment]
@ellis-johnson please rebase so we can merge.

**Review by darthhexx — APPROVED** [review]
lgtm

**Comment by darthhexx** [comment]
@facchettos are you all good with this PR as is? I'd like to get it merged so the follow up PR's can be merged as well.

**Comment by darthhexx** [comment]
Oh I see the changes requested were to files that mistakenly added to this PR, but they are actually in https://github.com/Azure/ARO-RP/pull/2587

File: pkg/portal/cluster/clusteroperators.go (hunk 2 above)
Comment (facchettos) [reply pkg/portal/cluster/clusteroperators.go]: haha no worries !

**Pull request merged**